api/manifest_test.go

```diff
@@ -71,34 +71,41 @@ func TestGetGitHubReleaseAssetForSHA(t *testing.T) {
 
 	client := mockGitHubClient{
 		Responses: map[string][]byte{
-			gitHubSHASearchURL(sha):          searchResults,
-			gitHubReleaseURL("merge_pr_123"): unsafeMarshal(releasesJSON),
+			gitHubSHASearchURL(fullSHA):      searchResults,
+			gitHubReleaseURL("merge_pr_123"): unsafeMarshal(releaseJSON),
 			downloadURL:                      data,
+			gitHubLatestReleaseURL:           unsafeMarshal(releaseJSON),
 		},
 	}
 
 	// 1) Data is unzipped.
-	manifest, err := getGitHubReleaseAssetForSHA(&client, sha)
+	manifest, err := getGitHubReleaseAssetForSHA(&client, fullSHA)
 	assert.Nil(t, err)
 	assert.Equal(t, []byte("magic data"), manifest)
 
+	// 1a) Release by empty SHA or "latest" match.
+	latestManifest, _ := getGitHubReleaseAssetForSHA(&client, "")
+	assert.Equal(t, manifest, latestManifest)
+	latestManifest, _ = getGitHubReleaseAssetForSHA(&client, "latest")
```

Comment:
I think this works as a test, but let me check my thinking. This shows that "" and "latest" are handled specially, because "foobar" would result in an error when we try to interpret it as a sha. But, since a manifest with fullSHA is in the (mock) response for any input, it doesn't show that "latest" and some other sha that isn't latest behave differently, that depends on https://api.github.com/repos/w3c/web-platform-tests/releases/latest being used as the URL, and we don't make any assertions about that. (That's all fine, not arguing for testing deeper.)

Reply:
lol. I'll split the cases, instead of being lazy.